pkg/kubectl/cmd/annotate.go

```diff
@@ -101,16 +100,8 @@ var (
 func NewCmdAnnotate(f cmdutil.Factory, out io.Writer) *cobra.Command {
 	options := &AnnotateOptions{}
 
-	// retrieve a list of handled resources from printer as valid args
-	validArgs, argAliases := []string{}, []string{}
-	p, err := f.Printer(nil, printers.PrintOptions{
-		ColumnLabels: []string{},
-	})
-	cmdutil.CheckErr(err)
-	if p != nil {
-		validArgs = p.HandledResources()
-		argAliases = kubectl.ResourceAliases(validArgs)
-	}
+	validArgs := []string{}
+	validArgs = cmdutil.ValidArgList(f)
```

Comment:
Just `validArgs := cmdutil.ValidArgList(f)` is enough.

Same as below.

Reply:
yes, thanks! will address that soon